app/helpers/common_parameters_helper.rb

```diff
@@ -11,12 +11,8 @@ def parameters_title
 
   def parameter_value_field(value)
     source_name = value[:source_name] ? "(#{value[:source_name]})" : nil
-    content_tag :div, :class => "form-group condensed" do
-      text_area_tag("value_#{value[:safe_value]}", value[:safe_value], :rows => (value[:safe_value].to_s.lines.count || 1 rescue 1),
-                    :class => "col-md-6", :disabled => true, :'data-hidden-value' => Parameter.hidden_value) +
-        fullscreen_button +
-        content_tag(:span, :class => "help-inline") { popover(_("Additional info"), _("<b>Source:</b> %{type} %{name}") % {:type => _(value[:source].to_s), :name => source_name})}
-    end
+    value_content_tag(value[:safe_value],[],value[:safe_value], [],
+                      popover('', _("<b>Source:</b> %{type} %{name}") % {:type => _(value[:source].to_s), :name => source_name}, 'data-placement' => 'top'))
```

Comment:
I'm a bit unsure whether these bold tags should be in there, can you extract the string in such a manner the tags are not included? Something like 
```ruby
'<b>' + _('Source') + ':</b> ' + _("%{type} %{name}") % ....
```
 @domcleal sorry to ping you but can you confirm? I think we have some html tags in translations but it's a bit weird when I see them in Transifex (at least for me).